src/router/index.tsx

```diff
@@ -15,6 +16,9 @@ export default function Routes() {
         <Route exact path="/">
           <Home />
         </Route>
+        <Route exact path="/product-list">
```

Comment:
```suggestion
        <Route path="/product-list">
```
🧐 Please recheck this comment. https://github.com/Cobda/cobda-web/pull/4#discussion_r563030869